travis/script.sh

```diff
@@ -72,20 +72,22 @@ if [ -z "${TEST}" ]; then
 
   # Set Helm settings for the broker
   case "${BROKER}" in
-      rabbitmq)
-	  HELM_BROKER_SETTINGS=" \
-	      --set redis.enabled=false \
-	      --set rabbitmq.enabled=true \
-	      --set celery.broker=rabbitmq \
-	  "
-	  ;;
-      redis)
-	  HELM_BROKER_SETTINGS=" \
-	      --set redis.enabled=true \
-	      --set rabbitmq.enabled=false \
-              --set celery.broker=redis \
-	  "
-	  ;;
+    rabbitmq)
+      HELM_BROKER_SETTINGS=" \
+        --set redis.enabled=false \
+        --set rabbitmq.enabled=true \
+        --set celery.broker=rabbitmq \
+        --set createRabbitMqSecret=true \
+    "
+    ;;
```

Comment:
Maybe missing an indent here? The travis jobs looks different than before.

Reply:
I have tried homogenizing the indents: was a mix of tabs and blanks, I have used only 2 blanks for indentation. bad idea? what looks different exactly?

Reply:
I provided a PR, but it seems it has nothing todo with travis...